README.md

```diff
@@ -11,55 +11,54 @@ define model structures using fields, trees and relations, and has a very
 flexible permission system which allows you to define who has access to
 perform actions under which limiting factors.
 
-eZ Publish exists in two versions; this, the community version is available under
+eZ Platform exists in two versions; this, the community version is available under
 the GPLv2 license, while several extended versions for enterprise & business is available
 under a more permissive business license, see [ez.no](http://ez.no/) for more info.
 
 ## Install, Upgrade and Getting started
-For installation & upgrade instructions, see [INSTALL.md](https://github.com/ezsystems/ezpublish-community/blob/master/INSTALL.md).
+For installation & upgrade instructions, see [INSTALL.md](https://github.com/ezsystems/ezplatform/blob/master/INSTALL.md).
 
-To get started with coding, see [GETTING_STARTED.md](https://github.com/ezsystems/ezpublish-community/blob/master/GETTING_STARTED.md).
+To get started with coding, see [GETTING_STARTED.md](https://github.com/ezsystems/ezplatform/blob/master/GETTING_STARTED.md).
```

Comment:
so, there will be a change in the repo name and ezpublish-community will gonna dissapear?